### PR TITLE
ISSUE-265 Added repos for Java and JSON

### PR DESCRIPTION
**Pull request by celandro**
Please consider merging the JSON generation and GAMEMASTER binaries into this project instead of
having it as a separate project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,3 +97,5 @@ If you don't want to compile POGOProtos but instead use it directly, check out t
 | Go               | https://github.com/pkmngo-odi/pogo-protos               |
 | Haskell          | https://github.com/relrod/pokemon-go-protobuf-types     |
 | Rust             | https://github.com/rockneurotiko/pokemon-go-protobuf-rs |
+| Java             | https://github.com/celandro/pogoprotos-java             |
+| JSON             | https://github.com/celandro/pokemongo-game-master       |
```
Comment (BrunnerLivio) [inline README.md]: Why use your fork and not [the source](https://github.com/BrunnerLivio/pokemongo-game-master)?

**Comment by celandro** [comment]
That code was mine and the code base appears to be a fork of my code. I'm
fine with having it all in one place though

**Comment by alucas** [comment]
We want to trust you, but it's strange.



Can you explain that ?

**Comment by BrunnerLivio** [comment]
@alucas @celandro 

Yes @celandro did write the main source code. He "copied" / "forked" it from [pokebattler](https://github.com/celandro/pokebattler-fight) (I think?). So he's the main contributor. I'm just the manager and the administrator of the repository.

But I don't see why use your fork? You would have to update it everytime a commit is submitted on my repo.

**Comment by celandro** [comment]
It was a complicated history. I had my source for months, he made a repo
and put just the jsons in, i forked that, added the readme and the java
source and linked to the new java proto library.

I would much prefer this be under AeonLucid and probably in python so it
would be more easily maintained.
Its pretty trivial, just have to read in the binary as a proto and use
https://github.com/google/protobuf/blob/master/python/google/protobuf/json_format.py
to convert to json.

I have some old binaries and json files sitting around in my pokebattler
repo for building out a historical record.

**Comment by BrunnerLivio** [comment]
@celandro I think thats @AeonLucid decision? I actually don't care if we merge it together or split it up.

But are you really currently referring to this PR? I see you sent the messages by email, so is it possible you think we're discussing #265 (Create GAME_MASTER.json), which is about merging pokemongo-game-master with POGOProtos together? In this PR you just added in the README the url to the JSON / Java repo

```
+| Java             | https://github.com/celandro/pogoprotos-java             |
+| JSON             | https://github.com/celandro/pokemongo-game-master       |
```

**Comment by celandro** [comment]
Yes I knew what the thread was about. I would prefer this NOT be merged for the JSON  repo and instead that be a directory of this project. I think that would make the most sense for the community and the python code to generate it is so trivial, it would be much better. This is not my decision of course.

The pogoprotos-java repo is good and clean, I will maintain it and that line should be added to the doc.

If the decision is not to include the json files as part of this repo, I prefer my repo as I wrote the code, the build process and the readme. I have been very quick on the json updates historically, same day, usually within a few hours (including rebuilding and testing my site with it).

**Comment by celandro** [comment]
@alucus Removed fork reference. I will be helping maintain the BrunnerLivio repo as well as maintaining the java protos.

I believe this can be merged

**Pull request closed without merging**